packages/cli/src/Server.ts

```diff
@@ -416,7 +422,7 @@ export class Server extends AbstractServer {
 
 		if (inDevelopment && process.env.N8N_DEV_RELOAD === 'true') {
 			const { reloadNodesAndCredentials } = await import('@/ReloadNodesAndCredentials');
-			await reloadNodesAndCredentials(this.loadNodesAndCredentials, this.nodeTypes, this.push);
+			await reloadNodesAndCredentials(this.loadNodesAndCredentials, this.push);
```

Comment:
`reloadNodesAndCredentials` cannot be a method in `LoadNodesAndCredentials`?

Reply:
done